src/css/styles.css

```diff
@@ -79,57 +101,186 @@ body {
     background-color: rgba(139, 227, 255, 0.46);
 }
 
+.pager__previous {
+    margin-right: 5px;
+}
+
+.pager__previous .fa {
+    margin-right: 0;
+}
+
+/* Change View */
+.change-view-actions {
+    border-bottom: 1px solid #eee;
+    border-top: 1px solid #eee;
+    margin: 1em 0 0.5em 0;
+    padding: 0.7em 0.2em;
+}
 
+.change-view-actions > div {
+    padding: 0;
+}
+
+.change-view-actions .btn-link {
+    border: none;
+    padding: 0;
+    font-size: inherit;
+    display: inline;
+    line-height: inherit;
+    vertical-align: baseline;
+}
+
+/* Version Selector */
+.version-selector {
+    display: flex;
+    justify-content: space-between;
+    margin: 0.5em 0;
+}
+
+.version-selector__item {
+    font-weight: normal;
+}
+
+.version-selector__item > span {
+    font-weight: bold;
+    margin-right: 0.5em;
+}
+/* END - Version Selector */
+
+/* ===== Annotation Form ===== */
+.lnk-action {
+    margin-right: 17px;
+}
 
 .annotation-form {
-    margin-bottom: 10px;
-    margin-top: 10px;
+    margin: 0.4em 0;
 }
 
-.annotation-form input[type="checkbox"] {
-    cursor: pointer;
-    margin-right: 7px;
+.annotation-form__signifiers {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 0.5em;
+    width: 100%;
 }
 
-.annotation-form label {
-    display: block;
-    font-weight: 200;
-    font-size: 1.4rem;
-    cursor: pointer;
+.annotation-form textarea {
+    border: 1px solid #c3c3c3;
+    border-radius: 3px;
+    color: #555;
+    height: 3em;
+    padding: 0.5em;
+    width: 100%;
 }
 
-.annotation-form--collapsed .info-text {
-    display: none;
+legend {
+    border: 0;
+    color: inherit;
+    font-size: 0.9em;
+    font-weight: 500;
+    margin: 0 0 0.5em 0;
 }
 
-.annotation-form--collapsed label {
-    float: left;
-    margin-right: 25px;
+.signifier-list, .signifier-list-item {
+    list-style: none;
+    margin: 0;
+    padding: 0;
 }
 
-@media (max-width: 990px) {
-    .annotation-form--collapsed div {
-        clear: both;
-        overflow: auto;
-    }
+.signifier-list {
+    display: flex;
+    flex-direction: column;
 }
 
-.container-list-view tbody tr {
+/* Remove margin from last signifier button  */
+.signifier-list > li:last-child {
+    margin: 0;
+}
+
+.signifier-list-item {
+    display: flex;
+    margin-right: 5px;
+}
+
+.annotation-form--collapsed .signifier-list {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    text-align: center;
+}
+
+/* Toggle buttons */
+.toggle {
+    opacity: 0;
+    width: 0;
+}
```

Comment:
This is definitely an improvement in that you can tab to the checkbox, but you still can’t actually see what checkbox you’re focused on. You might try adding a rule like:

```css
.toggle:focus + .toggle-btn::before {
  outline: 2px solid aliceblue;               // Generic case, whatever you want
  outline: -moz-mac-focusring auto 3px;       // Firefox on OS X (no Windows-specific equivalent, ¯\_(ツ)_/¯)
  outline: -webkit-focus-ring-color auto 5px; // WebKit + Blink
}
```

Obviously you can do whatever focus-specific styling you want; the above just gets you something close-ish to what the system does by default.